manifests/cf-manifest/manifest/010-cf-jobs.yml

```diff
@@ -222,6 +224,9 @@ jobs:
       - name: cf
     update:
       serial: true
+    properties:
+      bbs:
+        url: (( grab meta.bbs.api_location  ))
```

Comment:
What's happening when this runs on both database VMs? Do they both submit the same metric with the same name?